pages/demo/css/css-inline.tsx

```diff
@@ -5,6 +5,7 @@ import Header from '~/components/Header';
 import Footer from '~/components/Footer';
```

Comment:
Same as in the previous files.

We should only modify where to replace with `<FormattedText>`.